app/__init__.py

```diff
@@ -1,4 +1,5 @@
-from configs import Config
+from algoliasearch.search_client import SearchClient
+from configs import Config, search_user, search_key, search_resource_index
```

Comment:
Rather than importing in `search_user`, `search_key` and `search_resource_index`, why don't we go ahead and add those as constants to the `Config` class?
https://github.com/OperationCode/resources_api/blob/b6104234e3e5e2004b7cb285747815a44aaf29fc/configs.py#L43

Then we could just do `Config.SEARCH_USER` rather than importing them separately. This would be more in keeping with the config patterns of the app.

Reply:
Sounds good. 
I considered doing that but was hesitant to mess with that class since it was used with the create_app function.